modules/google-maps/maps-widget.php

```diff
@@ -128,6 +131,12 @@ public function form( $instance ) {
 		echo '	<label for="' . esc_attr( $zoom['id'] ) . '" class="wp-api-maps_zoom_label">' . esc_attr( 'Zoom:' ) . '</label>';
 		echo '	<input type="text" id="' . esc_attr( $zoom['id'] ) . '" name="' . esc_attr( $zoom['name'] ) . '" class="widefat" value="' . esc_attr( $zoom['val'] ) . '">';
 		echo '</p>';
+
+		// ScrollWheel option.
+		echo '<p>';
+		echo '	<input value="1" type="checkbox"' . checked( esc_attr( $scrollwheel['val'] ), 1, false ) . 'id="' . esc_attr( $scrollwheel['id'] ) . '" name="' . esc_attr( $scrollwheel['name'] ) . '" />';
+		echo '	<label for="' .  esc_attr( $scrollwheel['id'] ) . '">Disable Scroll Zoom</label>';
```

Comment:
__[code_sniffer]__
* Concat operator must be surrounded by a single space

<!-- Comment by [SideCI](https://sideci.com) -->